src/server/permissions/AppPermissions.ts

```diff
@@ -138,4 +142,6 @@ export const defaultPermissions: Array<IPermission> = [
     AppPermissions.apis.default,
     AppPermissions.moderation.read,
     AppPermissions.moderation.write,
+    AppPermissions.role.read,
+    AppPermissions.role.write,
```

Comment:
@Dnouv please remove these permissions from the default list. The default list shouldn't really change anymore, since it is used to _not break_ apps that were developed PRIOR to the permission system.

Reply:
Thank you! Resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/655